controls/radexpander/features/header-button-style.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Header Button Style
+page_title: Header Button Style
+description: This article will show how to customize the header button style of the RadExpander control.
+slug: radexpander-features-header-button-style
+tags: button style, header button, expander, expander button
+published: True
+position: 3
+---
+
+# HeaderButtonStyle
+
+The `RadExpander` control allows you to customize the style of its header, which is represented by a `RadToggleButton` element. To do so, you can set a new `Style` with `TargetType="RadToggleButton"` to the `HeaderButtonStyle` property of RadExpander.
+
+#### __[XAML] Setting a Style to the HeaderButtonStyle property__
+{{region radexpander-features-header-button-style-0}}
+    <telerik:RadExpander Header="My Expander">
+        <telerik:RadExpander.HeaderButtonStyle>
+            <!-- If you use NoXaml dlls set the BasedOn property of the Style: BasedOn="{StaticResource RadToggleButtonStyle}"--> 
+            <Style TargetType="telerik:RadToggleButton">
+                <Setter Property="Foreground" Value="Red"/>
+            </Style>
+        </telerik:RadExpander.HeaderButtonStyle>
+    </telerik:RadExpander>
+{{endregion}}
+
+__RadExpander with a custom HeaderButtonStyle__
+
+![](images/radexpander-features-header-button-style-0.png)
+
+## Retain the Default RadToggleButton Look
```

Comment:
I would suggest you to start this section with information about the loss of style. Currently, the section starts with "to keep the default style do that". I think it should be something like: "When the HeaderButtonStyle is set, the default style is completely overridden and its appearance fallbacks to the RadToggleButton's default styling. To keep the default look...".